index.js

```diff
@@ -56,6 +58,19 @@ DuplexWrapper.prototype._write = function _write(input, encoding, done) {
   this._writable.write(input, encoding, done);
 };
 
-DuplexWrapper.prototype._read = function _read(n) {
-  this._readable.resume();
+DuplexWrapper.prototype._read = function _read() {
+  if (this._shouldRead) return;
```

Comment:
> listening to the data event doesn't switch you into old-mode anymore

Interesting. Can we revert to https://github.com/deoxxa/duplexer2/blob/74fcd4394f3236e00932b6dc44da49bdec4fc055/index.js#L33-L37?

Anyway I just published v0.1.1. https://github.com/deoxxa/duplexer2/commit/298ce063bd8a8bdd778db04cac245f8febd03209